oresat_configs/constants.py

```diff
@@ -6,7 +6,10 @@
 
 from enum import IntEnum
 
-__version__ = "0.3.1"
+try:
+    from ._version import version as __version__  # type: ignore
```

Comment:
Was there a `mypy` error that you were getting without the `type: ignore` hint? I see that `_version.py` defines `version: str` so it should check out. I tried removing the hint and running `mypy` and it didn't turn up any errors on my end.

Reply:
Oh, I added when I tried to add it into a different file. Yea, it looks like it is not needed.

Reply:
I see the error now. I think I hit a bug with `pylama` caching `mypy` results?:

```bash
$ python -m build
...
Successfully built oresat-configs-0.3.2.dev10+g88b0fcb.d20240225.tar.gz and oresat_configs-0.3.2.dev10+g88b0fcb.d20240225-py3-none-any.whl
$ pylama
oresat_configs/_version.py:1: [C] C0114 Missing module docstring [pylint]
$ rm oresat_configs/_version.py 
$ pylama
$ pylama
$ mypy .
oresat_configs/constants.py:10: error: Cannot find implementation or library stub for module named "oresat_configs._version"  [import-not-found]
oresat_configs/constants.py:10: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
...
$ pylama
oresat_configs/constants.py:10: [E]  Cannot find implementation or library stub for module named "oresat_configs._version"  [import-not-found] [mypy]
oresat_configs/constants.py:10: [N]  See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports [mypy]
```
The `pylama` run version of `mypy` only picks up the error after I run `mypy` directly.  Very strange.